packages/@vuepress/theme-default/components/DropdownLink.vue

```diff
@@ -105,6 +116,11 @@ export default {
 
     isLastItemOfArray (item, array) {
       return last(array) === item
+    },
+
+    handleDropdown () {
+      const isTriggerByTab = event.detail === 0
```

Comment:
Added a reference here: https://developer.mozilla.org/en-US/docs/Web/API/UIEvent/detail, or the code is hard to maintain.

The `Tab + Click` is [UIEvent](https://developer.mozilla.org/en-US/docs/Web/API/UIEvent) > [KeyboardEvent](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent), so the detail is `0`.